Scripts/Runtime/Entities/Tasks/TaskFlow/TaskFlowDataSystem.cs

```diff
@@ -0,0 +1,24 @@
+namespace Anvil.Unity.DOTS.Entities
+{
+    /// <summary>
+    /// Data System (no update) for managing the world's <see cref="TaskFlowGraph"/>
+    /// </summary>
+    public partial class TaskFlowDataSystem : AbstractDataSystem
+    {
+        internal TaskFlowGraph TaskFlowGraph
+        {
+            get;
+        }
+
+        public TaskFlowDataSystem()
+        {
+            TaskFlowGraph = new TaskFlowGraph();
+        }
+
+        protected override void OnStartRunning()
+        {
+            base.OnStartRunning();
+            TaskFlowGraph.Harden();
```

Comment:
🚲 Consider building in a way that the graph is read by a factory type so there is no Harden/Bake/Render method. You'd instead do `TaskRelationshipFactory.Build(TaskFlowGraph)`

That way we can use the graph as the data and pass it into various different factories for debugging, visual rendering, etc... in the future. It also keeps the responsibilities separated.

Reply:
Added a TODO to rework this. I agree it's better to do it that way, just more work right now.